### PR TITLE
Add HTTR model to the VTB

### LINKED ISSUE #240: HTTR Core multiphysics RELAP-7/Griffin/BISON model
A steady state multiphysics core model was developed by the analysis team here at INL. 
We wish to contribute this model to the virtual test bed

it includes
- HTTR documentation
- Griffin homogenized core model
- Bison homogenized core conduction model
- BISON heterogeneous pin model
- RELAP-7 channel models for pins and control rod channels

### PR DESCRIPTION
**Pull request by GiudGiud**
closes #240

### CODE REVIEW

**Comment by GiudGiud** [comment]
@vincentlaboure if you want to have a last look before we release it

**Comment by vincentlaboure** [comment]
Will do, thanks.

**Review by vincentlaboure — CHANGES_REQUESTED** [review]
Looks good to me. Just a typo to fix.

File: doc/content/resources/codes_used.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ obtained through INL's [NCRC](https://inl.gov/ncrc/).
 
 - Heat Pipe Micro-Reactor [documentation](mrad/index.md) and [inputs](https://github.com/idaholab/virtual_test_bed/tree/main/microreactors/mrad)
 
+- HTTR Core multiphysics [documentation](httf/index.md) and [inputs](https://github.com/idaholab/virtual_test_bed/tree/main/htgr/httr/steady_state_and_null_transient)
```
[the remark this answers is not in the document]
Comment (GiudGiud) [reply doc/content/resources/codes_used.md]: done. good catch, thank you for taking a look

**Review by vincentlaboure — APPROVED** [review]

**Pull request merged**